build.rs

```diff
@@ -4,7 +4,38 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-pub const BINDINGS: &str = "bindings.rs";
+const BINDINGS: &str = "bindings.rs";
+
+#[cfg(feature = "gecko")]
+fn clang_args() -> Vec<String> {
+    use mozbuild::TOPOBJDIR;
+
+    let flags_path = TOPOBJDIR.join("netwerk/socket/neqo/extra-bindgen-flags");
+    println!("cargo:rerun-if-changed={}", flags_path.to_str().unwrap());
+
+    let mut flags: Vec<String> = std::fs::read_to_string(flags_path)
+        .expect("Failed to read extra-bindgen-flags file")
+        .split_whitespace()
+        .map(std::borrow::ToOwned::to_owned)
```

Comment:
Can you call `to_owned` directly on the `Iterator` returned by `split_whitespace()`?

```suggestion
        .to_owned()
```

https://doc.rust-lang.org/stable/std/str/struct.SplitWhitespace.html#impl-ToOwned-for-T

Reply:
This is used in a bunch of other places in gecko; might want to do a PR over there, too.

Reply:
If you feel strongly about it, let me know, happy to do it @larseggert.

If not, I would argue the overhead of producing a mozilla-central patch is not worth the small cleanup.